### PR TITLE
Feature/second milestone

### PR DESCRIPTION
**Pull request by Zappat0n**
- Add Display component
- Add Button component
- Add ButtonPanel component
- Add App component

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello  👋

**You've done pretty well here we just need to make a few changes and we are good to go. 👌**

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import PropTypes from 'prop-types';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+const App = ({ result }) => (
```
Comment (mikenath223) [inline src/components/App.js]: - [ ] App component shouldn't receive any props, please remove the result props passed to App.
- [ ] You can also leave the Display component without passing result props to it for now.

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      { result }
+    </p>
+  );
+};
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
+
+export default Display;
```
Comment (mikenath223) [inline src/components/Display.js]: - [ ] Please add default props to result it should have the default value = '0'

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 
Hi @Zappat0n, congratulations on this project :fireworks: Having removed the props from the App component :muscle: Having added the default props to the Display component :smile: I hereby approve the project :+1: 

Kindly merge into development, do not forget to submit the project completion form :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**